### PR TITLE
cincinnati: add quay secrets

### PR DESCRIPTION
**Pull request by steveej**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -10,7 +10,7 @@ applyTemplate:
 	oc process -f $(WHAT) | oc apply -f -
 .PHONY: applyTemplate
 
-postsubmit-update: prow-services origin-release ci-infra-imagestreams libpod prow-monitoring build-dashboards-validation-image image-mirror-setup knative-image-mirror-setup
+postsubmit-update: prow-services origin-release ci-infra-imagestreams libpod prow-monitoring build-dashboards-validation-image image-mirror-setup knative-image-mirror-setup cincinnati
```
Comment (stevekuznetsov) [inline Makefile]: does this target already exist?

File: projects/cincinnati/cincinnati.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: List
```
Comment (stevekuznetsov) [inline projects/cincinnati/cincinnati.yaml]: nit: no need for a list here

File: projects/cincinnati/cincinnati.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    labels:
+      ci.openshift.io/source-repo: cincinnati
+  spec:
+    successfulBuildsHistoryLimit: 1
+    failedBuildsHistoryLimit: 2
+    output:
+      to:
+        kind: ImageStreamTag
+        name: cincinnati-build-root:latest
+    strategy:
```
Comment (stevekuznetsov) [inline projects/cincinnati/cincinnati.yaml]: You need to specify the GitSource here

File: Makefile (hunk 1 above)
Comment (steveeJ) [reply Makefile]: It does now, thanks :)

File: projects/cincinnati/cincinnati.yaml (hunk 3 above)
Comment (steveeJ) [reply projects/cincinnati/cincinnati.yaml]: I'm not sure if I understand this right. please take another look

File: projects/cincinnati/cincinnati.yaml (hunk 2 above)
Comment (steveeJ) [reply projects/cincinnati/cincinnati.yaml]: the list is needed but I didn't have the `ImageStream` in the version you commented on

**Pull request merged**